dao/tx/tx.go

```diff
@@ -191,3 +192,32 @@ func (m *defaultTxModel) UpdateTxsStatusInTransact(tx *gorm.DB, blockTxStatus ma
 	}
 	return nil
 }
+
+func (m *defaultTxModel) GetLatestExecutedTx() (tx *Tx, err error) {
```

Comment:
Can you pass the `statuses` `txTypes` as parameters?
Currently, the name is very general, however there are filters in the function; the name does not match to the logics .

Reply:
OK, I will change it to GetlatestTx(txTypes []int, statuses []int64)